_static/custom.css

```
@@ -170,23 +181,141 @@ div.index-shields {
   }
 }
 
-@media screen and (max-width: 870px) {
+@media screen and (max-width: 875px) {
+  div.sphinxsidebar p.logo {
+    display: initial;
+  }
+
   .guide-container {
     flex-direction: column;
   }
+
   .guide-card {
     width: 100%;
     margin: 8px 0;
     padding: 0px;
   }
+
   .guide-card:not(:first-child) {
     margin-left: 0;
   }
+
   .guide-card ul {
     margin-left: 32px;
   }
+
+  img.component-image {
+    max-width: 80px;
+  }
 }
 
 .breadcrumbs li {
   display: inline;
 }
+
+a {
+  color: #004B6B;
+  text-decoration: none;
+  border-bottom: 1px dotted #004B6B;
+}
+
+a:hover {
+  color: #999999;
+  text-decoration: none;
+  border-bottom: 1px dotted #999999;
+}
+
+a:hover code {
+  background: none;
+}
+
+div.body p, div.body dd, div.body li, div.body blockquote {
+  hyphens: none;
+}
+
+/* dark theme */
+@media (prefers-color-scheme: dark) {
+
+  @media screen and (max-width: 875px) {
+    div.documentwrapper {
+      background-color: #212121;
+    }
+  }
+
+  body, div.body {
+    color: #ececec;
+    background-color: #212121;
+  }
+
+  .dark-invert, img.logo {
+    filter: invert(1);
+  }
+
+  .row-odd {
+    background-color: #282828;
+  }
+
+  table.docutils {
+    -webkit-box-shadow: 2px 2px 4px #3b3838;
+    box-shadow: 2px 2px 4px #3b3838;
+  }
+
+  div.body {
+    color: #ececec;
+    background-color: #212121;
+  }
+  
+  a, a.reference, a:visited {
+    color: #00bfff;
+    border-bottom: 1px dotted #00bfff;
+  }
+
+  a:hover {
+    color: #87ceeb;
+    border-bottom: 1px solid #87ceeb;
+  }
+
+  .toctree-wrapper a code {
+    color: #00bfff;
+  }
+
+  div.sphinxsidebar h3, div.sphinxsidebar h3 a {
+    color: #ececec;
+  }
+
+  div.sphinxsidebar ul {
+    color: #ececec;
+  }
+
+  div.sphinxsidebar a, .breadcrumbs a {
+    color: #ececec;
+    border-bottom: 1px dotted #ececec;
+  }
+
+  div.sphinxsidebar a:hover, .breadcrumbs a:hover {
+    border-bottom: 1px solid #ececec;
+  }
+
+  .guide-card {
+    background-color: #14504f;
```

I think keeping the guide just a lighter grey in dark mode fits better
```suggestion
    background-color: #393939;
```